docs/source/guides/VTFModding.rst

```diff
@@ -38,35 +86,81 @@ Though it is more a skins thing, making vtf skins is somewhat complicated, and t
 		"$nocull" "1"
 	}
 
-When we use vtf textures, we can only use the albedo and normal `maps <https://titanfall-skin-group.gitbook.io/titanfall-2-skin-creation/ms/genral-information/texture-maps>`__. Fire up `VTFEdit <https://nemstools.github.io/pages/VTFLib-Download.html>`__ and hit file, import, and grab your texture. then file, save as, and save it in the same folder as your .vmt. In your vmt, put the path to your texture in the parentheces after `"$basetexture"`, treating models as root. So i would put, `models\weapons_r2\vtfkraber\kraber_col`. Then do the same for your normal map, but when you import it, pick volume texture instead of animated texture. In `"$bumpmap"` put the path to your normal texture. Now create another vtf with literally any image. Put its path in `"$texture2"`. As far as i know, this is neccesary even though the texture isnt used. Your root folder should look somewhat like this::
+
+When we use vtf textures, we can only use the albedo and normal. Learn more about `texture maps <https://retryy.gitbook.io/tf2/wiki/create/texturemaps>`_ here.
+
+VTFEdit
+--------
+
+`VTFEdit`_ is a tool to edit, view, and create .vtf files.
+
+.. _VTFEdit: https://nemstools.github.io/pages/VTFLib-Download.html
+
+Launch `VTFEdit`_. Top left, click ``File``, ``Import``, find and Import your custom texture(s). 
+
+When importing your normal map. Choose to import as a ``Volume Map``
+When importing your diffuse map. Choose to import as a ``Animated Map``
+
+More info about .vtf format possibilities `here <https://retryy.gitbook.io/tf2/wiki/create/formats>`_, or the official source docs `here <https://developer.valvesoftware.com/wiki/Valve_Texture_Format>`_.
+
+After that, save your new .vtf's into the same folder as your custom .vmt with a simple name.
+
+Configuring your .vmt
+---------------------
+
+To know which texture/texture map you have. Analyze `this <https://retryy.gitbook.io/tf2/wiki/create/texturemaps>`_ wiki.
+
+In the ``"$basetexture"`` argument. You'll enter your .vtf texture directory. We'll use ``models\weapons_r2\car_ctm\NAMEOFVTF``. This is pointing to your custom diffuse .vtf with the simple name. Do not add the ``.vtf`` file extension to the end of the directory; ``models\weapons_r2\car_ctm\NAMEOFVTF.vtf``, or the game cannot find the texture.
+
+Do the same for your normal map in the ``"$bumpmap"`` argument.
+
+Now create another vtf with literally any image. Put its path in ``"$texture2"`` argument. As far as i know, this is necessary even though the texture isn't used. (i never had to do this but maybe you need it)
```

Comment:
```suggestion
In the ``"$basetexture"`` argument enter your .vtf texture directory. We'll use ``models\weapons_r2\car_ctm\NAMEOFVTF``. This should point to your custom diffuse .vtf with the simple name. The game expects these paths to be without the ``.vtf`` file extension - don't add it.

Do the same for adding your normal map with the ``"$bumpmap"`` argument.

In some cases you might have to create another vtf with literally any image. Put its path in the ``"$texture2"`` argument. As far as i know, this is sometimes necessary even though the texture isn't used.
```

A example of the finaly .vtm here might be needed, or a re-wording of the "arguments". Maybe using more common JSON terminology of keys?